src/content/docs/style-guide/capitalization/product-capability-feature-usage.mdx

```diff
@@ -1463,25 +1445,6 @@ We used to capitalize our platform components (below). However, we no longer pos
     </tr>
   </thead>
 
-  <tbody>
```

Comment:
Actually the logs are saying the mdx error is related to <tbody>. I think it's right here.

Reply:
related to `<tbody>`

Reply:
Great eye!